experiment-trait-data-visualizer/cache-refresh.R

```diff
@@ -0,0 +1,126 @@
+library(dplyr)
+library(tools)
+library(lubridate)
+options(scipen=999)
+
+# set up remote connection to BETYdb
```

Comment:
Sorry. Export works in bash, or Sys.setenv() in R. Hard coding it in is also okay, I used environment variables out of concern for leaving passwords in Github.